pkg/streamformatter/streamformatter.go

```diff
@@ -4,6 +4,8 @@ package streamformatter // import "github.com/docker/docker/pkg/streamformatter"
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/docker/docker/api"
+	"github.com/docker/docker/api/types"
```

Comment:
Haven't looked very closely yet, but CI may fail on this (`pkg` package importing API code); perhaps we should do the reverse (define the const here, and the API consuming it)

(these imports should also be in the second group below)

Reply:
so we have `api` to depend on `pkg`? I would expect the opposite is enforced